solana/modules/wormhole-anchor-sdk/src/wormhole/accounts.rs

```diff
@@ -1,4 +1,4 @@
-use anchor_lang::{prelude::*, solana_program};
+use anchor_lang::{prelude::*, solana_program,};
```

Comment:
nit: Usually for one-line imports, the trailing comma is removed
```suggestion
use anchor_lang::{prelude::*, solana_program};
```